app/views/home/home.html.erb

```diff
@@ -27,39 +27,12 @@
 </head>
 
 <body>
-
-    <!-- Navigation -->
-    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
-      <div class="container">
-        <%= link_to "PogHogs", '/', id: "logo" %>
-        <nav>
-          <ul class="nav navbar-nav navbar-right">
-            <li><%= link_to "Shop", items_path %></li>
-            <li class="dropdown">
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Categories <span class="caret"></span></a>
-              <ul class="dropdown-menu">
-                <% all_categories.each do |category| %>
-                  <li><%= link_to category.title, category_path(category) %></li>
-                <% end %>
-              </ul>
-            </li>
-            <%- unless current_user == nil -%>
-              <li><%= link_to "Log out", logout_path, method: :delete %> </li>
-            <%- else -%>
-              <li><%= link_to "Log In", login_path, method: :get %> </li>
-            <%- end -%>
-              <li><%= link_to "My Cart", '/cart' %></li>
-          </ul>
-        </nav>
-      </div>
-        <!-- /.container -->
-    </nav>
-
     <!-- Page Content -->
     <div class="container">
 
       <h1 class="home_title">PogHogs</h1>
       <h3 class="home_blurb">The premiere marketplace for pogs and pog accessories.</h3>
+      <h3><%= link_to "Shop Now!", items_path %></h3>
```

Comment:
added client requested link to all items on homepage, could use some better styling